src/analysis/retail/rogue/assassination/CHANGELOG.tsx

```diff
@@ -1,14 +1,16 @@
 import { change, date } from 'common/changelog';
-import { ToppleTheNun, Bigsxy} from 'CONTRIBUTORS';
+import { ToppleTheNun, Bigsxy, Whispyr} from 'CONTRIBUTORS';
 import SpellLink from 'interface/SpellLink';
 import SPELLS from 'common/SPELLS/rogue';
 import TALENTS from 'common/TALENTS/rogue';
 import SHARED_CHANGELOG from 'analysis/retail/rogue/shared/CHANGELOG';
 
 export default [
+  change(date(2023, 8, 24), `Improved support for Sepsis with Coooldown breakdown + Snapshoting`, Bigsxy),
```

Comment:
You could dual-attribute this change and remove the change further down:
```suggestion
  change(date(2023, 8, 24), `Improved support for Sepsis with Coooldown breakdown + Snapshoting`, [Bigsxy, Whispyr]),
```

Reply:
Got it, the earlier entry was there from when i branched @Whispyrs WIP, i wasnt sure if i should have kept it in or not.